src/gfn/modules.py

```diff
@@ -134,9 +134,71 @@ def to_probability_distribution(
 
 
 class ScalarEstimator(GFNModule):
+    r"""Class for estimating scalars such as LogZ.
```

Comment:
Note that `logZ` for unconditional TB is usually modeled with a single learnable parameter (`nn.Parameter`).
Should we consider modifying `ScalarEstimator` to support this kind of behavior?

Reply:
The GFNs themselves support this directly (you do not need to pass an estimator at all, instead you just pass a float for `Z`).

Reply:
This comment is because of `such as LogZ` in the docstring!

Reply:
I'm not entirely sure what would be most clear here but I'm open to suggestions.

Reply:
Why not just state flow functions of DB/SubTB??